Makefile

```diff
@@ -24,11 +26,19 @@ deploy:
 	@echo "> Fetching master branch and updating sources."
 	${remote} "${goto_src} && git fetch origin ${branch} && git checkout ${branch} && git reset --hard FETCH_HEAD"
 	${remote} "${goto_src} && pip install -r requirements.txt"
+	make rsync
+	@echo "> Launching gunicorn daemon."
 	${remote} "${goto_src} && pkill gunicorn; \
 		gunicorn --daemon -b 127.0.0.1:5000 preprod:app \
 		--error-logfile ~/log/preprod.log --access-logfile ~/log/preprod.log"
 	@echo "> App is deployed. Run \`make logs\` to follow activity."
 
+rsync:
+	@echo "> Synchronizing media from production server."
+	${remote} "rsync --archive --compress --human-readable --inplace --progress ${prod}:${prod_media_dir}/avatars/ ${src_dir}/cafebabel/uploads/users/"
```

Comment:
with `--delete` to remove any file that may have been deleted on the remote server.